rec_to_nwb/test/processing/device/test_deviceFactory.py

```diff
@@ -17,7 +19,7 @@
 
 class TestDeviceFactory(TestCase):
 
-    def test_factory_create_Device_successfully(self):
+    def test_device_factory_create_Device_successfully(self):
```

Comment:
this test does not make sense to me as we should not test if Mock creates ;)

Reply:
We mock input to factory and test output